src/main/java/com/foundrysoftware/minecraft/plugins/PluginTemplate.java

```diff
@@ -1,37 +1,171 @@
 package com.foundrysoftware.minecraft.plugins;
 
+import org.bukkit.Bukkit;
+import org.bukkit.Location;
+import org.bukkit.Material;
+import org.bukkit.World;
 import org.bukkit.command.Command;
 import org.bukkit.command.CommandSender;
-import org.bukkit.entity.Player;
+import org.bukkit.entity.*;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.entity.EntityShootBowEvent;
+import org.bukkit.event.entity.ProjectileHitEvent;
+import org.bukkit.event.player.PlayerInteractEvent;
+import org.bukkit.inventory.Inventory;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.inventory.PlayerInventory;
+import org.bukkit.inventory.meta.ItemMeta;
+import org.bukkit.inventory.meta.PotionMeta;
 import org.bukkit.plugin.java.JavaPlugin;
+import org.bukkit.potion.PotionData;
+import org.bukkit.potion.PotionEffect;
+import org.bukkit.potion.PotionEffectType;
 
 import javax.validation.constraints.NotNull;
 
-public class PluginTemplate extends JavaPlugin {
+import java.util.*;
+import java.util.List;
+import java.util.logging.Logger;
+import com.sun.org.apache.bcel.internal.generic.InstructionConstants;
+import org.bukkit.potion.PotionType;
+
+import java.util.HashMap;
+import java.util.*;
+import java.util.logging.Logger;
 
+public class PluginTemplate extends JavaPlugin {
+    private static final String ARROW_TYPE_TNT = "tnt";
     private final String PLUGIN_NAME = "PluginTemplate";
 
-    public void onEnable(){
-        getLogger().info("["+ PLUGIN_NAME +"] Started ");
+    public void onEnable() {
+        getLogger().info("[" + PLUGIN_NAME + "] Started ");
     }
 
-    public void onReload(){
-        getLogger().info("["+ PLUGIN_NAME +"] Server Reload");
+    public void onReload() {
+        getLogger().info("[" + PLUGIN_NAME + "] Server Reload");
     }
 
-    public void onDisable(){
-        getLogger().info("["+ PLUGIN_NAME +"] Server Stopping");
+    public void onDisable() {
+        getLogger().info("[" + PLUGIN_NAME + "] Server Stopping");
     }
 
     @Override
     public boolean onCommand(@NotNull CommandSender sender, @NotNull Command command, @NotNull String label, String[] args) {
-        if (label.equalsIgnoreCase("simple")){
-            if (sender instanceof Player){
-                Player me = (Player)sender;
-                me.sendMessage("Plugin Working");
+        String simple="simple";
+        if (label.equalsIgnoreCase(simple)) {
+            if (sender instanceof Player) {
+                Player me = (Player) sender;
+                String message = "Plugin Working";
+                me.sendMessage(message);
                 return true;
             }
+            if (label.equalsIgnoreCase("hawkeye")) {
```

Comment:
Issue #1 **ALL** String literals should be constants unless there is a good reason